content/en/contribute/code/releasing.md

```diff
@@ -10,7 +10,34 @@ aliases: >
 
 ## CHT Core
 
-[Create an issue](https://github.com/medic/cht-core/issues/new/choose) for either a Major/Minor or Patch release issue template and assign it to the release manager to follow the process.
+A release is a set of changes that is planned to be used by at least one implementation that uses the CHT.
+
+### Release Process
+
+The high-level process for a release is as follows:
+
+* The team sees an opportunity they want to go after. The opportunity addresses a need of at least one implementation that uses the CHT and will be used by that implementation after the release.
+* The team agrees on a solution for it.
+* Tickets are added to the [Product Team Activities board](https://github.com/orgs/medic/projects/134/views/3) for what's being built.
+* A [release manager](#release-manager) is assigned from the team.
+* The release manager [creates an issue](https://github.com/medic/cht-core/issues/new/choose) for either a [Major/Minor or Patch](#major-minor-patch-release) release issue template and follows the process.
+* Change is built by a developer together with [quality assistance]({{% ref "/contribute/medic/product-development-process/quality-assistance" %}}).
+* [Code is reviewed]({{% ref "/contribute/code/workflow#code-reviews" %}}).
+* Code is merged.
+* Code is released.
```

Comment:
I read this page and this section in particular to be instructions on what the release process is. With that context though, this line would seem to be a recursive reference. More likely though I have misunderstood the intention of the page & section. How about if the section title is "Building & Releasing Software"?

Reply:
Yes, `code` is better and consistent with the rest of the steps. Thank you

Reply:
How about `Building & Releasing CHT Core Changes`?